packages/@react-facet/core/src/mapFacets/mapFacets.ts

```diff
@@ -8,22 +8,24 @@ export function mapFacetsLightweight<M>(
   facets: Facet<unknown>[],
   fn: (...value: unknown[]) => M | NoValue,
   equalityCheck?: EqualityCheck<M>,
+  onCleanup?: () => void,
```

Comment:
It looks like this was added at one point, but its no longer needed?

Reply:
Because of [this needed change](https://github.com/Mojang/ore-ui/pull/125/files#diff-ff6794aab3570cdf0ed8365e8fb2120fb9e3c70bbb4edfbbd0f7336489ee9bffR52-R54) to cleanup the new cache for SharedFacets in `useSharedSelector` I had to change alll functions similar to `mapFacetsCached` to include an optional onCleanup callback function to allow the cleaning up the cache when the facets are no longer there.